src/main/java/by/vadzimmatsiushonak/bank/api/model/dto/response/ConfirmationResponse.java

```diff
@@ -0,0 +1,11 @@
+package by.vadzimmatsiushonak.bank.api.model.dto.response;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class ConfirmationResponse {
+
+    @ApiModelProperty(value = "Indicates the bank payment Confirmation was successful.", example = "true")
```

Comment:
if it's related only to Bank Payment we need to change name to BankPaymentConfirmationResponse
Otherwise we need to change field description value to something common